.build/main-app/app/scripts/built.js

```diff
@@ -77,10 +105,19 @@ angular.module('tombola.snapCardGame',[
 
 (function () {
     'use strict';
-    angular.module('tombola.snapCardGame')
+    angular.module('Tombola.SnapCardGame')
+
+        .controller('SnapCardController',['$scope','GameModel','DeckService',function ($scope, GameModel, DeckService){
```

Comment:
space after each of the comma's here please, you've done it in places but not consistently